package.json

```diff
@@ -50,7 +50,7 @@
     "cheerio": "1.0.0-rc.2",
     "xmldom": "0.1.27",
     "logops": "2.1.0",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#task/lazyAttsNgsiv2-pr762",
```

Comment:
Revert before merging this PR, once https://github.com/telefonicaid/iotagent-node-lib/pull/813 get merged.

Reply:
Fixed in 869cb3d